src/main/resources/org/archive/commons.properties

```diff
@@ -0,0 +1,5 @@
+operator=BnF
+publisher=Bibliotheque nationale de France
+wat.warcinfo.description=Description du format WAT
```

Comment:
My apologies for answering you so late. I can see three scenarios here :

**Scenario 1** : Use IIPC as default operator and publisher, with a small description for WAT WARC-Info
```
operator=IIPC
publisher=International Internet Preservation Consortium
wat.warcinfo.description=Web Archive Transformation (WAT) is a specification for structuring metadata generated by Web crawls. Specification here : https://webarchive.jira.com/wiki/display/Iresearch/Web+Archive+Transformation+%28WAT%29+Specification,+Utilities,+and+Usage+Overview
```

**Scenario 2** : abstract generic operator and publisher, but keeping the WAT WARC-Info description
```
operator=<operator name here>
publisher=<publisher name here>
wat.warcinfo.description=Web Archive Transformation (WAT) is a specification for structuring metadata generated by Web crawls. Specification here : https://webarchive.jira.com/wiki/display/Iresearch/Web+Archive+Transformation+%28WAT%29+Specification,+Utilities,+and+Usage+Overview
```

**Scenario 3** : full abstract generic parameters
```
operator=<operator name here>
publisher=<publisher name here>
wat.warcinfo.description=<WAT WARC-Info description here>
```